stdlib/oauth2/src/main/ballerina/src/oauth2/outbound_oauth2_provider.bal

```diff
@@ -338,11 +346,11 @@ function getAuthTokenForOAuth2ClientCredentialsGrant(ClientCredentialsGrantConfi
     }
 }
 
-# Process the auth token for OAuth2 direct token mode.
+# Process the OAuth2 token for direct token mode.
 #
-# + grantTypeConfig - Direct token configurations
+# + grantTypeConfig - Direct token mode configurations
 # + oauth2CacheEntry - OAuth2 cache entry
-# + return - Auth token or `Error` if an error occurred during the HTTP client invocation or validation
+# + return -OAuth2 token or `oauth2:Error` if an error occurred during the HTTP client invocation or validation
```

Comment:
```suggestion
# + return -OAuth2 token or `oauth2:Error` if an error occurred during the HTTP client invocation or validation
```
Please refer the template for the correct format.

Reply:
```suggestion
# + return -OAuth2 token, or else `oauth2:Error` if an error occurred while HTTP client invocation or validation
```